package.json

```diff
@@ -24,26 +24,26 @@
     },
     "dependencies": {
         "glob": "^7.1.6",
-        "qs": "^6.10.1",
-        "ts-morph": "^20.0.0"
+        "qs": "^6.12.1",
+        "ts-morph": "^22.0.0"
     },
     "devDependencies": {
         "@types/glob": "^7.1.3",
-        "@types/jest": "^29.5.1",
-        "@types/node": "^14.14.35",
-        "@types/qs": "^6.9.6",
+        "@types/jest": "^29.5.12",
+        "@types/node": "^20.12.11",
```

Comment:
I guess it would be good idea to have consistent versions used here and in CI  https://github.com/HLTech/pact-gen-ts/blob/0e0c42a46d5b78ca7afce63ef81da172571a568a/.github/workflows/node.js.yml#L15